concent_api/core/views.py

```diff
@@ -346,21 +466,30 @@ def store_message_and_message_status(golem_message_type, task_id, raw_golem_mess
     golem_message.full_clean()
     golem_message.save()
 
-    if status == ReceiveStatus:
-        receive_message_status  = ReceiveStatus(
+    if status is not None:
+        receive_message_status  = status(
             message     = golem_message,
-            timestamp   = message_timestamp
+            timestamp   = message_timestamp,
+            delivered   = delivered
         )
         receive_message_status.full_clean()
         receive_message_status.save()
 
-    elif status == ReceiveOutOfBandStatus:
-        receive_out_of_band_status = ReceiveOutOfBandStatus(
-            message     = golem_message,
-            timestamp   = message_timestamp,
-            delivered   = True
-        )
-        receive_out_of_band_status.full_clean()
-        receive_out_of_band_status.save()
-    else:
-        return None
+    return None
+
+
+def check_file_status(file_transfer_token):
```

Comment:
Please add a type annotation here that `file_transfer_token` is of type `FileTransferToken`.